src/contoroller/user/user.js

```diff
@@ -8,70 +8,81 @@ class UserController {
   }
   //ok
   async create(req, res) {
-    //Validte received data to create a new user
+    //Validting data received in body of request
     const { error } = this.validateUser(req.body);
     if (error) return res.status(400).send(error.message);
-    //Check user existance
+    //De-structuring req.body
+    let { username, password, email } = req.body;
+    //Checking user existance with email
     let user = await this.User.findOne({
-      where: { email: req.body.email },
+      where: { email: email },
     });
     if (user) return res.status(400).send("User already registered.");
-    //Destructure req.body
-    let { username, password, email } = req.body;
-
-    //hassh the password
+    //hashing the password
     const salt = await bcrypt.genSalt(10);
     password = await bcrypt.hash(password, salt);
-    //Save user to the DB
+    //Save new user to the DB
     user = await this.User.create({
       username: username,
       password: password,
       email: email,
     });
+    //Generatting token
     const token = user.generateAuthToken();
     user.token = token;
```

Comment:
This line does not make any sense here.

1. after setting a property on a Sequelize model instance you need to save the instance. Read the docs [here](https://sequelize.org/docs/v6/core-concepts/model-instances/#saving-only-some-fields).
2. Your User model does not have a `token` field. You need to add a new column to your users table and store the token in that column. The token can be stored in the `generateAuthToken` after the token gets created.